docs/FingerprintApi.md

```diff
@@ -379,7 +379,7 @@ public class FingerprintApiExample {
 
 ## searchEvents
 
-> SearchEventsResponse searchEvents(limit, visitorId, bot, ipAddress, linkedId, start, end, reverse, suspect)
+> SearchEventsResponse searchEvents(limit, paginationKey, visitorId, bot, ipAddress, linkedId, start, end, reverse, suspect)
```

Comment:
Shouldn't this be just `searchEvents(limit, optionalParams)` ?

Reply:
Fixed